scripts/release/rpm/Dockerfile.mariner

```diff
@@ -0,0 +1,24 @@
+ARG tag=1.0
+
+FROM cblmariner.azurecr.io/base/core:${tag} AS build-env
+ARG cli_version=dev
+
+RUN tdnf update -y
+RUN tdnf install -y rpm-build gcc libffi-devel python3-devel openssl-devel make diffutils patch dos2unix python3-virtualenv perl
+
+WORKDIR /azure-cli
+
+COPY . .
+
+RUN dos2unix ./scripts/release/rpm/azure-cli.spec && \
+    REPO_PATH=$(pwd) CLI_VERSION=$cli_version rpmbuild -v -bb --clean scripts/release/rpm/azure-cli.spec && \
+    cp /usr/src/mariner/RPMS/x86_64/azure-cli-${cli_version}-1.x86_64.rpm /azure-cli-dev.rpm
```

Comment:
For CentOS, it is `/root/rpmbuild/RPMS/x86_64/azure-cli-${cli_version}-1.*.x86_64.rpm`. Interesting difference.